packages/styled-components/src/models/Keyframes.js

```diff
@@ -18,11 +19,7 @@ export default class Keyframes {
 
   inject = (styleSheet: StyleSheet) => {
     if (!styleSheet.hasNameForId(this.id, this.name)) {
-      styleSheet.insertRules(
-        this.id,
-        this.name,
-        styleSheet.options.stringifier(...this.stringifyArgs)
-      );
+      styleSheet.insertRules(this.id, this.name, masterStylis(...this.stringifyArgs));
```

Comment:
We don't hash keyframes, so just having it use the OG unmodified stylis instance